### PR TITLE
Chris/styling and tooltip

### LINKED ISSUE #104: Create tool-tips on mouse-over bubbles
## Acceptance Criteria
This story can be considered done when:

**Given** a view of the front-end which shows bubbles representing frequency of meetings with a certain person / government office / organization,

**When** I move the mouse over a bubble in the UI

**Then** it should display a tooltip which shows what/who that particular bubble refers to, and how many meetings there were.

### PR DESCRIPTION
**Pull request by chrisyeoward**
Each **Pull Request** should relate to a single feature, therefore a single **Issue**. The smaller the **Pull Request** the faster it can be reviewed & merged. It should not be merged by the author.

## Issues to close
Added tooltips, added styling to page.

Issues to close: #104, #72 

## Notes

Notes on the **Pull Request**. 

For example: *Run database schema migrations first.*

## Tasks

* [ ] Documentation (inc. diagrams) - create/update accordingly

### CODE REVIEW

File: src/js/frontend/src/components/About.js [hunk 1]
```diff
@@ -19,11 +19,16 @@ class About extends React.Component{
         <div style={[
           styles.aboutContent,
         ]}>
+        <a href='#' onClick={this.props.closeFlyover}><img style={[styles.close]}src={closeAboutData}/></a>
```
Comment (elischutze) [inline src/js/frontend/src/components/About.js]: style police: leave a space btwn `}` and `src`

File: src/js/frontend/src/components/NavItem.js [hunk 2]
```diff
@@ -10,26 +10,47 @@ class NavItem extends React.Component{
...
-      </li></a>
+        <a style={[
+          styles.a,
+        ]}
+        target={target}
+        onClick={this.props.onClick}
+        href={this.props.href}>
+          <img style={[
+            styles.img,
+          ]} height='30px' src={this.props.iconData} />
```
Comment (elischutze) [inline src/js/frontend/src/components/NavItem.js]: could height be part of `styles.img`?

File: src/js/frontend/src/components/d3Chart.js [hunk 3]
```diff
@@ -152,10 +152,33 @@ var d3Chart = {
     //   .domain([])
     //   .range();
 
+    var tooltip = d3.select("body").append("div")
+                .attr("class", "tooltip")
+                .style("opacity", 0);
+
+    var tooltip
```
Comment (elischutze) [inline src/js/frontend/src/components/d3Chart.js]: style police: let's keep semicolon usage consistent

File: src/js/frontend/stylesheets/main.less [hunk 4]
```diff
@@ -26,19 +23,83 @@ ul, ol, p {
...
+}
+
 g.bubble text {
   text-anchor: middle;
   fill: white;
+  dominant-baseline: central;
+  pointer-events: none;
+  font-family: "Roboto Slab", Arial, sans-serif;
+  font-weight: lighter;
+  // font-weight: lighter;
```
Comment (elischutze) [inline src/js/frontend/stylesheets/main.less]: remove duplicate line

**Comment by aspiers** [comment]
Cool, thanks! Only glanced through this and tested quickly on my local setup but seems to do the job! Not 100% convinced about white text on pink though - can be slightly hard to read.

**Comment by chrisyeoward** [comment]
Yeah I was thinking about the contrast too, but thought it looked ok on my screen. I've also changed the font size in the bubbles to scale with bubble size, and for the smallest ones you've still got the tooltip, so think it's alright.

**Comment by chrisyeoward** [comment]
Made requested changes Eli, ready to go!

**Review by elischutze — APPROVED** [review]

**Pull request merged**